src/addAllAssetsToCompilation.js

```diff
@@ -31,7 +31,16 @@ function resolveOutput(compilation, addedFilename, outputPath) {
 async function addFileToAssets(
   compilation,
   htmlPluginData,
-  { filepath, typeOfAsset = 'js', includeSourcemap = true, hash = false, publicPath, outputPath, files = [] }
+  {
+    filepath,
+    typeOfAsset = 'js',
+    includeSourcemap = true,
+    includeGzipped = true,
```

Comment:
Hehe that might just be a difference of opinion, but I think the default behavior would be good if it silently skipped missing files. Because it's very easy to look in the directories and see what files are there, so if they're not there, then you know you need to change something in your webpack config. And especially if both of these have `true` as the default value, I would assume that it means: "copy them if they exist, otherwise ignore." That seems like a sane default to me.

But either way works for me